src/common/config.cc

```diff
@@ -712,6 +708,25 @@ void md_config_t::set_val_or_die(const char *key, const char *val)
   assert(ret == 0);
 }
 
+struct is_integer_member : public boost::static_visitor<bool> {
+    template<typename T,
+             typename boost::enable_if<boost::is_integral<T>, int>::type = 0>
+    bool operator()(const T md_config_t::* /* member_ptr */) const {
+        return true;
+    }
+    template<typename T,
+             typename boost::enable_if_c<!boost::is_integral<T>::value, int>::type = 0>
+    bool operator()(const T md_config_t::* /* member_ptr */) const {
+        return false;
+    }
+};
+
+bool md_config_t::config_option::is_safe() const {
+  return safe ||
```

Comment:
What I am trying to say is that it makes the code harder to read when you insert a comment line in the middle of a statement:

```
return xyz ||

  // now let's talk about things 
  abc ||

 // some more things to say
 tuv;
```